guides/RELEASES.md

```diff
@@ -6,59 +6,62 @@
+* Move the ticket to the 'PR' column
 
 ## 'PR' column
 
 * **Get approval:** get the PR peer-reviewed
   * if any code or other issues are identified, apply fixes to the feature branch
 * **Merge** the approved feature branch into the `develop` branch
   * do **not** merge using Github :warning:
-* **Ship it** - the updated `develop` branch should be auto-deployed to the *develop* environment
+* **Ship it** - the updated `develop` branch should be auto-deployed to the *sandbox* environment
   * ensure that shipping was successful:
-    * in CI (`develop-ship-it` job was successful for the right commit)
-    * in _consul_ (for the expected commit/version and health)
+    * in [CI](https://concourse.dp-ci.aws.onsdigital.uk/) (`sandbox-ship-it` job was successful for the right commit)
+    * in [_consul_](https://consul.dp.aws.onsdigital.uk/ui/eu/services) (for the expected commit/version and health)
 * Developer moves the story to `Testing` column
 
 ## 'Testing' column
-* Developer **tests** the feature in the *develop* environment
-  * similar to approval, above: any fixes go into your branch for re-approval, re-merge, etc
+* Developer **tests** the feature in the *sandbox* environment
+  * similar to approval, above: any fixes go into your feature branch for re-approval, re-merge, etc
+    * if your feature branch was deleted (manually or automated by github settings), you can always reopen it via the github UI
 * Developer moves the story to `PO sign off` (if required) or `Ready for release` column
 
 ## 'PO sign off' column
 
-* Product owner **reviews changes** in the *develop* environment
-  * Bug fixes applied to feature branch (and re-approved, etc into the `develop` branch)
+* Product owner **reviews changes** in the *sandbox* environment
+  * bug fixes applied to feature branch (and re-approved, etc into the `develop` branch)
 * **Sign off:** the product owner moves the story to `Ready for release` column
 
 ## 'Ready for release' column
 
 * **Create Release branch** (e.g. `release/1.7.0` see [version control](VERSIONING.md) ) taken from `develop` branch
 * **Create a PR** - to merge the release branch into the `main` branch
+  * the release PR must include the release notes e.g. the features, bugs that have been made
   * any issues (e.g. in CI) are applied to release branch
-  * Release branch fixes regularly merged back to `develop`
+  * release branch fixes regularly merged back to `develop`
 * **Merge** into `main` branch
 * [Create Release tag](TAGS.md) from `main` branch
+  * tag must be published as a release, with bullet point list of changes in the release notes. These should be obtained from the release pull request description
   * release branch can be deleted
 
 ## Deployment
 
-* the `main` branch requires **manual deployment** to the *production* environment - this can be done in CI
+* The `main` branch requires **manual deployment** to the *production* environment - this can be done in CI
```

Comment:
Might want to review the use of `main` here as most of our apps are still on `master`

Reply:
So line 7 does state that main could be master in some repos, Im wary of changing this and then someone having to change it back when we do finally move all apps to use main branch